File: packages/boozsoft-auth/src/main/java/org/boozsoft/repo/stock/ArApCodeRepository.java
package org.boozsoft.repo.stock;

import org.boozsoft.domain.entity.stock.ArApCode;
import org.springframework.data.repository.reactive.ReactiveCrudRepository;
import reactor.core.publisher.Flux;

public interface ArApCodeRepository extends ReactiveCrudRepository<ArApCode, String> {

    Flux<ArApCode> findAllByBillStyleOrderById(String billStyle);

}

File: packages/boozsoft-platform-zhongzhang/src/main/kotlin/org/boozsoft/repo/stock/ArApCodeRepository.kt
package org.boozsoft.repo.stock

import org.boozsoft.domain.entity.stock.ArApCode
import org.springframework.data.repository.reactive.ReactiveCrudRepository
import reactor.core.publisher.Flux

interface ArApCodeRepository : ReactiveCrudRepository<ArApCode, String> {
    fun findAllByBillStyleOrderById(billStyle: String): Flux<ArApCode>
}